tests/components/dynalite/test_bridge.py

```diff
@@ -1,136 +1,101 @@
 """Test Dynalite bridge."""
-from unittest.mock import Mock, call, patch
+from unittest.mock import Mock, call
 
+from asynctest import patch
 from dynalite_lib import CONF_ALL
 import pytest
 
-from homeassistant.components.dynalite import DATA_CONFIGS, DOMAIN
-from homeassistant.components.dynalite.bridge import BridgeError, DynaliteBridge
+from homeassistant.components import dynalite
 
-from tests.common import mock_coro
 
-
-async def test_bridge_setup():
-    """Test a successful setup."""
+@pytest.fixture
+def dyn_bridge():
+    """Define a basic mock bridge."""
     hass = Mock()
-    entry = Mock()
     host = "1.2.3.4"
-    entry.data = {"host": host}
-    hass.data = {DOMAIN: {DATA_CONFIGS: {host: {}}}}
-    dyn_bridge = DynaliteBridge(hass, entry)
+    hass.data = {
+        dynalite.DOMAIN: {dynalite.DATA_CONFIGS: {host: {dynalite.CONF_NOWAIT: True}}}
+    }
+    bridge = dynalite.DynaliteBridge(hass, host)
+    return bridge
+
+
+async def test_bridge_setup(dyn_bridge):
```

Comment:
If we don't patch our bridge code in the init, config flow and light tests, we will already have tested all the bridge code. So then we don't need these extra tests in this module.

Reply:
correct. removed them